lib/config.js

```diff
@@ -1,14 +1,21 @@
 /*!
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-import bedrock from 'bedrock';
-const {config} = bedrock;
+import {config} from '@bedrock/core';
 
 const namespace = 'bedrock-did-resolver-http';
 const cfg = config[namespace] = {};
 
-// support did:key and veres one keys by default
-cfg.supportedMethods = ['key', 'v1'];
+// a list of which did:methods we supported
+// experimental is for tests only
+cfg.supportedMethods = ['key', 'v1', 'experimental'];
+// FIXME this can be expanded to support Multikey or JsonWebKey2020
+cfg.publicKeyFormats = [
+  'Ed25519VerificationKey2020',
+  'X25519KeyAgreementKey2020',
+  // this is strictly for testing
+  'ExperimentalVerificationKey2022'
```

Comment:
```suggestion
  'ExperimentalVerificationKey2022'
```

Same problem here... this is not a valid verification key.

Reply:
it's required for test data so the issue is we need to setup a did-resolver for the tests and then setup a did resolver for other use cases.

Reply:
Marking this as resolved as we use `Ed25519VerificationKey2018` now.